maintenance/importConfig.ts

```diff
@@ -34,7 +35,91 @@ import * as fs from "fs-extra";
 import * as JSON5 from "json5";
 import * as path from "path";
 import * as qs from "querystring";
+import { promisify } from "util";
 import xml2json from "xml2json";
+import yargs from "yargs";
+
+const execPromise = promisify(child.exec);
+
+yargs
+	.option("source", {
+		description: "source of the import",
+		alias: "s",
+		type: "array",
+		choices: ["oh", "ozw"], // oh: openhub, ozw: openzwave
+		default: ["oh"],
+	})
+	.option("ids", {
+		description: "devices ids to download",
+		type: "array",
+	})
+	.option("download", {
+		alias: "D",
+		description: "Download devices DB from <source>",
+		type: "boolean",
+		default: false,
+	})
+	.option("clean", {
+		alias: "C",
+		description: "Clean temprorary directory",
+		type: "boolean",
+		default: false,
+	})
+	.option("manufacturers", {
+		alias: "m",
+		description: "Parse and update manufacturers.json",
+		type: "boolean",
+		default: false,
+	})
+	.option("index", {
+		alias: "i",
+		description: "Parse and update devices index.json",
+		type: "boolean",
+		default: false,
+	})
+	.option("devices", {
+		alias: "d",
+		description: "Parse and update devices configurations",
+		type: "boolean",
+		default: false,
+	})
+	.example(
+		"import -s ozw -Dmid",
```

Comment:
Would it make sense to merge the source and download options into one?

Reply:
I don't understand, what do you mean?

Reply:
You have a `download` option `-D` and a `source` option `-s`. 
As far as I understand, `download` requires that you provide the source and `source` without `download` does nothing.

Reply:
Ehhh - nevermind. The import logic is also different for both sources.

Reply:
> As far as I understand, download requires that you provide the source and source without download does nothing.
Why not just remove the download option and make the script download first when there's a source given?